language/tools/move-cli/src/package.rs

```diff
@@ -105,18 +67,30 @@ impl<'a> SourceFilter<'a> {
     }
 }
 
-struct MovePackage {
+pub struct MovePackage {
     /// Name of the package
-    name: &'static str,
+    name: String,
     /// The directory containing all the .move source files
     sources: Vec<SourceFilter<'static>>,
     /// Dependencies
     deps: Vec<&'static Lazy<MovePackage>>,
```

Comment:
Almost certainly yes, but I didn't want to take on refactoring `MovePackage` too much in this PR, since it wasn't related to the CLI restructuring I was interested in. Curious to hear thoughts of @mengxu-fb .

Reply:
Yes, `deps: Vec<impl AsRef<MovePackage>>` feels better. But it does not have to go into this PR. I'll remind myself to get a PR for this `MovePackage` change after the landing of this one.